doc/03-Configuration.md

```diff
@@ -61,6 +61,45 @@ For this you need an `ApiUser` object with at least the following permissions on
     also configure the secondary master's API command transport.
     Icinga DB Web then uses this transport if the primary one is not available.
 
+## General Configuration
+
+Some default values for actions with dialogs in the UI can be adjusted (e.g. While acknowledging a problem).
```

Comment:
Was something wrong with my suggestion?

You're now talking of *some values* and in the next sentence of *these options*. I'd wonder, which options are meant.

And while there are indeed *actions **without** dialogs*, I don't think this nuance needs to be mentioned here.

Though, I see why you added *some* at the start, as I used *particular* to refer to dialogs myself.

So what about:

```suggestion
You can adjust some default values of options users have while interacting with particular dialogs in the UI. (e.g. While acknowledging a problem)
```